AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift

```diff
@@ -0,0 +1,161 @@
+//  Created by Anna Olak on 24/02/2021.
+//
+import AirCastingStyling
+import SwiftUI
+
+struct CreateSessionDetailsView: View {
+    
+    @EnvironmentObject private var sessionContext: CreateSessionContext
+    @StateObject var viewModel: CreateSessionDetailsViewModel
```

Comment:
As always, I don't like this. There was a discussion about that couple of times 😛 Apple doesn't recommend that: (see Apple's dev comment here --> https://developer.apple.com/forums/thread/650618)

Ania found a nice way to be sure that the viewModel won't be created a couple of times that with `init(wrappedValue: )`. 
Tbh, I'm not 100% sure if `init(wrappedValue: )` isn't a default init used here, you can test it. If yes, it should be ok, if not it's dangerous.

Reply:
I didn't know about that. As long as it is the view that is created every time we are going to create a new session I think it won't be a problem in that case. As I am seeing it right now is that it is created once and doesn't recreate when I move out of that view. Maybe I've not tested the way I should. Please tell me how to be sure and what do you think I should do now?

Reply:
if you tested it and it's not created more than once while being on this view, it should be fine. :)
Anyway, I think it's good to know that it's not recommended :D

Reply:
For sure it is! Thanks for letting me know, I will try to take care in the future from now

Reply:
As I understand, view model (or any other source of truth for data) should be marked as @StateObject in the place where it's initialized, and then as @ObservedObject in the view to which it is passed (and which should change with a change of some published property). So for example, when we were working on the chart with @pawelgil, we've initialized ChartViewModel in SessionCardView and marked in as @StateObject and then we've passed it to ChartView, where it was an @ObservedObject. This way the whole chart view wasn't reinitialized with each change of ChartViewModel.

Reply:
Thank You for the whole, great explanation. I will do my best to make it great and as you said and merge